pkg/apis/istio/v1alpha1/session_types.go

```diff
@@ -8,12 +8,27 @@ import (
 
```

Comment:
Maybe we can have a make target doing that for us? (and make it as default make target)

In other words: "shall we make make making those changes on our behalf as part of default make's make" ;)

Reply:
Not sure what this is referencing?

Reply:
weird, comment on line 7:

`// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file`

Reply:
we do that allready. make codegen :) It will regen the other code in this package based on this file

Reply:
right. :man_facepalming: